recipe/patches/0004-fix_CHECK_LIBRARY_EXISTS.patch

```diff
@@ -15,7 +15,7 @@ index b01832c..97b1b0d 100644
  ENDIF(MSVC)
  
 -CHECK_LIBRARY_EXISTS(${HDF5_C_LIBRARY_hdf5} H5free_memory "" HAVE_H5FREE_MEMORY)
-+CHECK_LIBRARY_EXISTS(${HDF5_C_LIBRARIES} H5free_memory "" HAVE_H5FREE_MEMORY)
++# CHECK_LIBRARY_EXISTS(${HDF5_C_LIBRARIES} H5free_memory "" HAVE_H5FREE_MEMORY)
```

Comment:
Do we want to set this to True in our patch?

Reply:
@hmaarrfk, It seems like that probably makes sense but I also think you understand this variable better than I do.  Make whatever change you think makes sense as far as I'm concerned.  Feel free to merge the PR once you're happy.